test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -2095,6 +2121,7 @@ def expected_postgres_references_manifest(self, model_database=None):
                 'model.test.ephemeral_copy': ['source.test.my_source.my_table'],
                 'model.test.ephemeral_summary': ['model.test.ephemeral_copy'],
                 'model.test.view_summary': ['model.test.ephemeral_summary'],
+                'exposure.test.notebook_exposure': [],
```

Comment:
Not sure why the test still claims to be getting a reference to model.test.view_summary since I removed it 🤔  

```
E   AssertionError: {'model.test.view_summary': ['exposure.test.notebook_e[319 chars]: []} != {'model.test.ephemeral_copy': ['model.test.ephemeral_s[344 chars]py']}
E   - {'exposure.test.notebook_exposure': [],
E   + {'exposure.test.notebook_exposure': ['model.test.view_summary'],
E   ?                                      +++++++++++++++++++++++++
E   
```

Reply:
Since this is the `parent_map`, you _should_ include `view_summary` here, since it's a parent of `notebook_exposure`:
```suggestion
                'exposure.test.notebook_exposure': ['model.test.view_summary'],
```